server/src/main/java/au/org/aodn/ogcapi/server/core/model/ExtendedCollection.java

```diff
@@ -0,0 +1,17 @@
+package au.org.aodn.ogcapi.server.core.model;
+
+import au.org.aodn.ogcapi.features.model.Collection;
+import lombok.Getter;
+import lombok.Setter;
+
+
+/**
+ * This class is extended from Collection, and used to add more fields to the Collection class.
+ */
+@Setter
+@Getter
+public class ExtendedCollection extends Collection {
+
+    private SummariesModel summaries;
```

Comment:
Can we not use a new class but Map<String, Object>?

Have you add new test case ?

Reply:
btw name is as properties not summaries

Reply:
got it. Maybe for be consistent with frontend, a simple string field is better.